build/postinstall.sh.cmake

```diff
@@ -201,9 +128,9 @@ extract_debug_symbols()
 mecho --blue "## Extracting debug info from files and saving them into dbg"
 
 # strip out debug symbols into external file
-case "$COMPILEFOR" in
-	apple)
-		echo "extraction of debug symbols for $COMPILEFOR currently not supported" >&2
+case "$SYSTEM_NAME" in
+	Apple)
+		echo "extraction of debug symbols for Apple currently not supported" >&2
 		i686-apple-darwin10-strip -S ${DESTDIR}bin/s25client
```

Comment:
perhaps we should use cmake to find the corresponding strip too and use it here?